IBAnimatableTests/ColorTypeTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ColorTypeTests.swift
+//  IBAnimatable
+//
+//  Created by Steven on 4/26/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import XCTest
+@testable import IBAnimatable
```

Comment:
That's right, if we have internal things to test, you need `@testable`, but I was thinking that we don't have any, do we?